datadog/dogstatsd/base.py

```diff
@@ -24,9 +24,13 @@
     # pypy has the same module, but capitalized.
     import Queue as queue  # type: ignore[no-redef]
 
+# pylint: disable=unused-import
```

Comment:
NIT can we resolve/remove this? If an import is unused, I think we can omit it?

Reply:
Its used in the comments and they give me a lint issue when I removed them

Reply:
Spoke offline. seems like the linter wasn't running on this file before.